notebooks/model_tutorial/1b_model_training_sensorium+.ipynb

```diff
@@ -8,7 +8,7 @@
     }
```

Comment:
<p>Same here with the comment vs markdown file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>fixed</p>